### ISSUE TITLE
[1.18.2] Incompatibility with Immersive Weathering and Botania, gives strange repeating tooltips

### ISSUE THREAD

**Issue by hamOverlord** [fzzyhmstrs/EMI_loot#5]
I'm not entirely sure what the link between these three mods are but when playing with Emi-Loot and Immersive Weathering installed, weapons have a tooltip that says "Killed by , Killed by , or Killed by" repeated twice

![image](https://user-images.githubusercontent.com/30599923/204118623-52b9b2ef-88e2-4470-bec5-f013a4d9d8b2.png)

Tested with only the following mods installed:
```
emi_loot-0.3.1+1.18.2.jar
emi-0.4.3+1.18.2.jar
fabric-api-0.67.0+1.18.2.jar
immersive_weathering-1.18.2-3.0.2-fabric.jar
```

If I replace Immersive Weathering with Botania it has a similar result, and it appears to be repeating the string "that is Manasteel Shears, Shears or Elementium Shears"

![image](https://user-images.githubusercontent.com/30599923/204118758-eabc4899-d05c-4d35-9889-935d031d074a.png)

```
Botania-1.18.2-435-FABRIC.jar
emi_loot-0.3.1+1.18.2.jar
emi-0.4.3+1.18.2.jar
fabric-api-0.67.0+1.18.2.jar
Patchouli-1.18.2-71.1-FABRIC.jar
trinkets-3.3.1.jar
```

**Comment by fzzyhmstrs** [comment]
Thank you for the notice! Quite a strange result you are seeing, as I don't think I interact with the normal tooltips at all.. or at least don't remember where I did that. Are there any other notable types of repeating tooltip? Or just the killed by and botania one?

**Comment by hamOverlord** [comment]
As far as I can tell its just the killed by and botania one. I made a decently sized modpack (~130 mods) and botania was the first mod that I removed (since the message mentioned manasteel) and then it changed to the killed by message. I removed and readded a lot of different of combinations of mods and no other messages other than the killed by one showed up which makes me think its only those two

If it helps I remember seeing killed by as an entry in your lang file. It's really strange because you'd expect it to show a string from Immersive Weathering like it did with Botania

**Comment by fzzyhmstrs** [comment]
Thanks for investigating that on your end a bit more! I'm pretty stumped as to what could be causing that, as EMI Loot doesn't interact directly with tooltips at all. 

I have been able to replicate this on my end, so there's definitely something going on. That said; as far as I can tell, this is unique to 1.18.2. I _wasn't_ able to replicate on 1.19.2.

@AstralOrdana , strange interaction between IW and EL on 1.18.2 here. Any insight you can give from your end? This only happens when IW is active, without it the weapon/equipment tooltips are like normal.

Something seems to be takingthe `LiteralText.EMPTY` in the ItemStack tooltip that makes the blank space before "When in Main Hand" and replacing it with a random combination of pieces from my lang file.

**Comment by fzzyhmstrs** [comment]
I've ruled out Cloth Config and ModMenu, both of with are jar-in-jar with IW

**Comment by fzzyhmstrs** [comment]
Victory!! This was caused by the way 1.18.2 handles EMPTY texts (with a singleton). I have a method that appends to that, which was affecting the EMPTY object. In the next update I'll have a fix for it that adds a `copy()` so the original EMPTY isn't affected.

![image](https://user-images.githubusercontent.com/72876796/204408853-534ecec2-d7e5-4f52-9053-1362e608579c.png)

**Comment by fzzyhmstrs** [comment]
Fixed in EMI_Loot 0.3.1a+1.18.2

**Comment by hamOverlord** [comment]
Ayyy awesome!! I'm glad you figured it out!
